scripts/linters/custom_eslint_checks/run_custom_eslint_tests.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
```

Comment:
Move this to the scripts dir (This is a driver script right?)

Reply:
Done.